macos.md

```diff
@@ -24,11 +24,17 @@
    <img src="./macos-4.1-homebrew-next-steps.png"><br><br>
 6. Copy each line in the following text, paste it in the terminal and hit return.<br><br>
    ```bash
-   brew install flyctl git node@18 python yarn
+   brew install flyctl git node@18 python
    brew link --overwrite node@18
    ```
-   This uses Homebrew to install `flyctl`, Git, Node.js, Python and Yarn.<br><br>
+   This uses Homebrew to install `flyctl`, Git, Node.js, Python.<br><br>
 7. Copy each line in the following text, paste it in the terminal and hit return.<br><br>
+   ```bash
+   brew install corepack
+   corepack prepare pnpm@latest --activate
+   ```
+   This uses Homebrew to install `corepack`, and uses Corepack to install `pnpm`.<br><br>
```

Comment:
this also needs to be edited if you change the instructions above to remove the Homebrew command

Reply:
Changed to your suggestion
[297d87c](https://github.com/upleveled/system-setup/pull/24/commits/297d87c25ac2b035a4a6c7107baa0da8faa4844c)